common/api/editor-common.api.md

```diff
@@ -19,6 +19,8 @@ import { TransformProps } from '@itwin/core-geometry';
 
 // @beta
 export interface BasicManipulationCommandIpc extends EditCommandIpc {
+    // (undocumented)
+    addAllowedChannels(channelKeys: string[]): Promise<IModelStatus>;
```

Comment:
i'm never sure where people will want their logic for initializing an editing session, since they'll always have a backend and a frontend. I suppose there may be cases where it could be on the frontend. In that case, you should add `removeAllowedChannels` too.

Reply:
I think these functions should only allow a single channel name.